testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -284,6 +287,27 @@ public void testPredicateReflectedInExplain()
                 "(predicate|filterPredicate|constraint).{0,10}(nationkey|NATIONKEY)");
     }
 
+    @Test(dataProvider = "sortOrders")
+    public void testSortItemsReflectedInExplain(SortOrder sortOrder)
+    {
+        // Even if the sort items are pushed down into the table scan, it should still be reflected in EXPLAIN (via ConnectorTableHandle.toString)
+        @Language("RegExp") String expectedPattern = hasBehavior(SUPPORTS_TOPN_PUSHDOWN) ?
+                "sortOrder=\\[nationkey:\\w+:\\w+ " + sortOrder + "] limit=5" :
+                "\\[5 by \\(nationkey " + sortOrder + "\\)]";
```

Comment:
i don't know what the code style says, but i like to put `?` and `:` and the beginning of the line for symmetry